doc/src/explore/tutorials.md

```diff
@@ -21,7 +21,7 @@ the `wallet` command used in the remainder of this tutorial in your path.
 Simply leave the terminal with Sui running and start a new terminal for the
 remainder of this tutorial.
 
-## Gather accounts and gas objects
+## Gather accounts and Sui objects
```

Comment:
I would not do it this way. We use the term "Sui objects" quite pervasively to mean any object in Sui. Calling gas objects "Sui objects" as well will create an enormous amount of confusion.

I'd suggest leaving "gas" in these general descriptions and talk about "SUI" only if it matters (e.g., if we refer to the actual code in SUI.move).

Reply:
Thanks, Adam. I'm not finding any instances of "gas" that need replacement given the Wallet commands and output retain the term.

Instead, I'm trying to disambiguate in the glossary and will likely abandon this PR:
https://github.com/MystenLabs/sui/pull/1134

Please advise. Thanks.

Reply:
I think it's a good idea. I guess my tagging @mystenadmin and @adeabiodun on this PR did not have the desired effect, but I would still suggest asking them about their view on the gas naming strategy in the docs and conversations about Sui.

Reply:
> Agreed that we should use SUI to refer to the Sui token. Wherever we say "gas objects", we should instead say "SUI tokens" or "SUI coins".

Whoops. Sorry i missed this, Sam. I was afraid of introducing confusion my mixing and matching, after speaking with Adam.

Does it make sense to rename this as "Gather accounts and SUI tokens" given the remainder of that section and doc use "gas objects" or some variant? Should we use SUI tokens/coins more liberally throughout?

Thanks for your guidance!